tofu/imas2tofu/_core.py

```diff
@@ -625,7 +625,8 @@ def _rhosign(rho, theta):
 
     def __init__(self, preset=None, dids=None, ids=None, occ=None, idd=None,
                  shot=None, run=None, refshot=None, refrun=None,
-                 user=None, tokamak=None, version=None, get=None, ref=True):
+                 user=None, tokamak=None, version=None,
+                 ids_base=None, synthdiag=None, get=None, ref=True):
```

Comment:
For future devs (and yourself), it could be a nice to add a bit of doc

Reply:
Good point, will add docstr in future PR
Thanks !